source/slang/slang-emit-hlsl.cpp

```
@@ -29,6 +29,77 @@ double _slang_asdouble(uint64_t x)
 }
 )";
 
+static int32_t _mapSlangCoopVecComponentTypeToHLSL(
+    int32_t slangValue,
+    bool inputInterpretationIsPacked)
```

Cppcheck (2.20.0)

[information] Include file

(missingInclude)